cli/core/pkg/command/config.go

```diff
@@ -216,39 +216,29 @@ var initConfigCmd = &cobra.Command{
 		if cfg.ClientOptions.CLI == nil {
 			cfg.ClientOptions.CLI = &configapi.CLIOptions{}
 		}
-		repos := cfg.ClientOptions.CLI.Repositories
-		finalRepos := []configapi.PluginRepository{}
-		for _, repo := range config.DefaultRepositories {
-			var exists bool
-			for _, r := range repos {
-				if repo.GCPPluginRepository.Name == r.GCPPluginRepository.Name {
-					finalRepos = append(finalRepos, r)
-					exists = true
-				}
-			}
-			if !exists {
-				finalRepos = append(finalRepos, repo)
-			}
+
+		serverName := ""
+		server, err := cfg.GetCurrentServer()
+		if err == nil && server != nil {
+			serverName = server.Name
 		}
-		cfg.ClientOptions.CLI.Repositories = finalRepos
```

Comment:
We no longer need this I gather?
Is it ok if the user turns off context-aware discovery, or maybe we don't support that at all?

Reply:
We don't support it at all now. And this is kind of a dead code that I removed to improve the readability of that function.

Reply:
Great